iati/guidance_and_support/models.py

```diff
@@ -98,3 +99,22 @@ class KnowledgebasePage(AbstractContentPage):
     """A base for a single Knowledgebase page."""
 
     subpage_types = []
+
+
+class CommunityPage(DefaultPageHeaderImageMixin, AbstractIndexPage):
+    """A base for the Community page."""
+
+    parent_page_types = ['home.HomePage']
+    subpage_types = []
+
+    text_box = models.TextField(max_length=255, null=True, blank=True, help_text='A small ammount of text explaing what IATI-Discuss is.')
```

Comment:
Generalise the help text here, maybe to something like: 'A small amount of text to describe the community". This is in case we use a different tool instead of Discuss in the future, or the page is used for something else.